src/components/formFieldArray.jsx

```diff
@@ -0,0 +1,31 @@
+import {FieldArray} from "formik";
+import {FormField} from "@/components/formField";
+
+const FormFieldArray = ({ name, label, fields, values, itemStructure }) => {
+    return <div>
+        <label>{label}</label>
+        <FieldArray name={name}>
+            {({ remove, push }) => (
+                <div className>
+                    {values.map((item, index) => (
+                        <div key={index} className={"border border-slate-300 mb-3 p-5 rounded-md"}>
```

Comment:
It's not a good practice to use index a value for the key: 
![Uploading image.png…]()